chore(Table, Table Page): Document caption requirement & add inline sorting controls demo (#2646)
Amsterdam/design-system: packages/css/src/components/table/README.md

# Table

> Presents related values in rows and columns, from a compact reference table to a large data set users need to scan, sort, or filter.

Author: > from a compact reference table

This reads a bit confusing because these kinds of simple tables are primarily not meant for the compact theme.

Maybe:

```suggestion
Presents related values in rows and columns: whether it is a simple reference table or a large data set to inspect, sort and filter.
```
